theme/mixins/backdrop.less

```diff
@@ -7,6 +7,11 @@
   transition: opacity 0.3s;
   z-index: @z-index-backdrop;
   opacity: 0;
+  @media @desktop {
+    // This is required as otherwise on desktop the backdrop
+    // Overflows the page.
+    max-width: 100vw;
```

Comment:
This mixin also defines `position: fixed` and `left: -999px`. Wouldn't this mean that it doesn't actually cover the full width of the screen on desktop anymore?